var/spack/repos/builtin/packages/3dtk/package.py

```diff
@@ -37,7 +37,7 @@ class _3dtk(CMakePackage):
     depends_on('cmake@3.5:', when='@trunk', type='build')
     depends_on('cmake@2.6.1:2', when='@1.2', type='build')
     depends_on('ninja', type='build')
-    depends_on('boost+serialization+graph+regex+filesystem+system+thread+date_time+program_options')
+    depends_on('boost@:1.75+serialization+graph+regex+filesystem+system+thread+date_time+program_options')
```

Comment:
Looks good, but was wondering if the constraint is documented anywhere? If so can you add a link as a comment above this directive?

Reply:
I am not aware of it being documented anywhere. I discovered the limit as I was trying to build it.